packages/gasket-request/README.md

```diff
@@ -0,0 +1,135 @@
+# @gasket/request
+
+The purpose of this package is to provide a consistent request object for Gasket
+plugins and apps to use, regardless of the request handling framework.
+
+## Installation
+
+#### Existing apps
+
+```shell
+npm install @gasket/request
+```
+
+## Functions
+
+### GasketRequest
+
+The `GasketRequest` class is a representation of a request object that can be
+used by Gasket plugins and apps. It is a consistent shape that can be used
+across different request handling frameworks.
+
+A `GasketRequest` object has the following properties:
+
+| Property | Type   | Description       | Arguments     |
+|----------|--------|-------------------|---------------|
+| headers  | object | Request headers   | required      |
+| cookies  | object | Request cookies   | default: `{}` |
+| query    | object | Query parameters  | default: `{}` |
+| path     | string | Request path      | default: `''` |
+
+
+The only required property to create a `GasketRequest` is the `headers` which
```

Comment:
Probably don't need this note for end-users since creation always goes through `makeGasketRequest` in the public interface.